recipes/tmap/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "tmap" %}
+{% set version = "0.1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/reymond-group/tmap/archive/9aa043dd1af022161048ee0a4b5554ec5e34977f.zip
+  sha256: 56cf4f83d663950ab7ef41deb425b8e1c01de378fb5239aeef7b6613a6edf212
+
+  patches:
+    - 01_bind2d_deprecated.diff
+    - 02_setup_windows.diff
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  binary_relocation: False  # [osx]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.1
+    - make  # [not win]
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+  host:
+    - ninja  # [win]
```

Comment:
```suggestion
    - ninja  # [win]
  host:
```

Reply:
Ah, I hadn't notice https://github.com/conda-forge/ninja-feedstock/pull/22 was finally merged! Thanks, yes, the `ninja` can be in `requirements: build` with no issue.